### PR TITLE
lnd: v0.13.1 rc1 branch prep

### PR DESCRIPTION
**Pull request by Roasbeef**
This PR updates the release notes ahead of the upcoming tag of `v0.13.1-beta.rc1`

### CODE REVIEW

File: lntest/itest/log_error_whitelist.txt [hunk 1]
```diff
@@ -284,3 +284,4 @@
 <time> [ERR] INVC: SettleHodlInvoice with preimage <hex>: invoice already canceled
 <time> [ERR] RPCS: [/invoicesrpc.Invoices/SettleInvoice]: invoice already canceled
 <time> [ERR] HSWC: ChannelLink(<chan>): outgoing htlc(<hex>) has insufficient fee: expected 33000, got 1020
+<time> [ERR] RPCS: [/lnrpc.Lightning/CloseChannel]: rpc error: code = Canceled desc = context canceled
```
Comment (yyforyongyu) [inline lntest/itest/log_error_whitelist.txt:287]: Interesting. I too saw this line popped up yesterday, something must have changed to cause it?

Comment (Roasbeef) [reply lntest/itest/log_error_whitelist.txt:287]: Not sure...seeing it pop up more often lately. One of those things where we should stop and consider if the error white list is actually carrying out its intended goals...in this case though, this isn't a newly added error log as it's from the unary RPC interceptor. 

If I had to guess, it seems we're trying to close a channel but then the test ends successfully as we detect the channel close before the RPC client gets the final response back?

Comment (yyforyongyu) [reply lntest/itest/log_error_whitelist.txt:287]: Haha I thought about the whitelist too just yesterday, seems like it's better to ignore error loggings as long as the tests pass.

> If I had to guess, it seems we're trying to close a channel but then the test ends successfully as we detect the channel close before the RPC client gets the final response back?

Yeah I would guess so.

File: docs/release-notes/release-notes-0.13.1.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ Scripts received as part of an upfront shutdown script [are now properly
...
 widespread relay of potential cooperative channel closures.
 
+## Wallet Unlocking
+
+[A new flag has been added](https://github.com/lightningnetwork/lnd/pull/5457)
+to permit users to set the new password file config while at the same time,
+allowing `lnd` to start up and accept new wallet initialization/creation via the
+RPC interface. The new wallet-unlock-allow-create option instructs lnd to not
+fail if no wallet exists yet but instead spin up its unlocker RPC as it would
+without the wallet-unlock-password-file being present.  This is not recommended
```
Comment (yyforyongyu) [inline docs/release-notes/release-notes-0.13.1.md:16]:
```suggestion
without the wallet-unlock-password-file being present. This is not recommended
```

File: docs/release-notes/release-notes-0.13.1.md [hunk 3]
```diff
@@ -56,4 +74,46 @@ reverted](https://github.com/lightningnetwork/lnd/pull/5404) as it
 introduced a regression that would cause payment failure due to mismatching
 heights.
 
+[A bug has been fixed that would previously cause any HTLCs settled using the
+`HltcInteceptor` API calls to always be re-forwarded on start
+up](https://github.com/lightningnetwork/lnd/pull/5280).
+
+[A bug has been fixed in the parameter parsing for the `lncli` PSBT funding
+call](https://github.com/lightningnetwork/lnd/pull/5441).  Prior to this bug
```
Comment (yyforyongyu) [inline docs/release-notes/release-notes-0.13.1.md:82]:
```suggestion
call](https://github.com/lightningnetwork/lnd/pull/5441). Prior to this bug
```

**Review by yyforyongyu — APPROVED** [review]
LGTM⚡️

File: docs/release-notes/release-notes-0.13.1.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ Scripts received as part of an upfront shutdown script [are now properly
 sanitized](https://github.com/lightningnetwork/lnd/pull/5369) to ensure
 widespread relay of potential cooperative channel closures.
 
+## Wallet Unlocking
+
+[A new flag has been added](https://github.com/lightningnetwork/lnd/pull/5457)
```
Comment (guggero) [inline docs/release-notes/release-notes-0.13.1.md:11]: Oh, I forgot about the release notes. Should we perhaps add a check to the GitHub actions that the release notes were updated?

**Review by guggero — APPROVED** [review]
Thanks for adding the notes for the PRs I forgot! LGTM :100:

File: docs/release-notes/release-notes-0.13.1.md (hunk 3 above)
Comment (Roasbeef) [reply docs/release-notes/release-notes-0.13.1.md:82]: It's a vim line folding artefact lol

File: docs/release-notes/release-notes-0.13.1.md (hunk 4 above)
Comment (Roasbeef) [reply docs/release-notes/release-notes-0.13.1.md:11]: Yeh it's been on my TOOD list for a while, but I haven't got around to finalizing the hacky bash script I started on lol. Thinking minimally it should: fail the build if based on the milestone, and the release notes file for that milestone, the PR number isn't in the raw markdown. So minimally you just need to mention the PR itself, but ideally the contributor adds more detailed information about the change.

**Pull request merged**